sdks/python/setup.py

```diff
@@ -102,6 +102,8 @@ def get_version():
   'google-apitools>=0.5.6,<1.0.0',
   'proto-google-cloud-datastore-v1==0.90.0',
   'googledatastore==7.0.0',
+  # GCP packages required by tests
+  'google-cloud-bigquery>=0.22.1,<1.0.0',
```

Comment:
done.

https://issues.apache.org/jira/browse/BEAM-1583 for tracking this discussion.